CMakeLists.txt

```
@@ -83,8 +76,9 @@ target_link_libraries(roboteam_utils
 target_compile_options(roboteam_utils PRIVATE
         --std=c++17
         -Wall
-#        -Werror
         -Wextra
+        -pedantic-errors
+        -Werror=pedantic
```

```suggestion
        -pedantic-errors
        -Werror=pedantic
```

Don't you mean
-Werror 
-pedantic
?

---

Ah yeah, I was testing some gcc flags. I looked at the doc and -Werror -pedantic makes more sense.
These things are NOT the same though, see https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#Warning-Options

---

As in, these things are legitimate flags, but I meant to change them back.